package.json

```diff
@@ -43,6 +60,7 @@
     "test:eslint": "./node_modules/.bin/eslint \"./packages/*/src/**/*.?(js|jsx)\"",
     "test:mocha": "f () { lerna exec --scope=@deriv/${1:-'*'} -- npm run test:mocha ;}; f",
     "test:stylelint": "stylelint \"./packages/*/src/**/*.s(a|c)ss\"",
+    "test:jest": "jest --all",
```

Comment:
is there a way to run `test:jest` for a specific package? and should this be included in the base `test` on line 59?

Reply:
It is, `npm run test:jest` accepts params, so we can do `npm run test:jest  packages/trader` or even the test file itself.

Reply:
Ok great 👍

Reply:
And should `npm run test:jest` be a part of `test` on line 59?